caffeine/src/main/java/com/github/benmanes/caffeine/cache/BoundedLocalCache.java

```diff
@@ -3156,7 +3166,8 @@ Object writeReplace() {
       @Override public long getMaximum() {
         return cache.maximum();
       }
-      @Override public void setMaximum(long maximum) {
+      @Override public void setMaximum(long maximum){ setMaximum(maximum, PERCENT_MAIN);}
```

Comment:
whitespace.  However, I suspect their coding style is to put onto distinct lines rather than all on one?

Reply:
Hah! You are right and I got tricked by IntelliJ folding!